### PR TITLE
Standfirst jsx

### PR DESCRIPTION
**Pull request by davidpmccormick**
Converts Standfirst to .jsx and deletes the .njk version

screen shot 2018-04-05 at 17 20 38

### CODE REVIEW

File: common/views/components/Index/index.js [hunk 1]
```diff
@@ -4,29 +4,30 @@ import ChapterIndicator from '../ChapterIndicator/ChapterIndicator';
 import CopyUrl from '../CopyUrl/CopyUrl';
 import Divider from '../Divider/Divider';
 import EventScheduleItem from '../EventScheduleItem/EventScheduleItem';
```
Comment (davidpmccormick) [inline common/views/components/Index/index.js]: These changes are just a reshuffle for everything to be alphabetised.

**Comment by jamesgorrie** [comment]
Like it.

As we're talking about creating concepts - could we stick this into a category of body slices (or something)?

Body parts I enjoy, but I wonder if it makes it harder to understand?

**Comment by davidpmccormick** [comment]
How about just a `body` category?

**Comment by davidpmccormick** [comment]
Don't think so. I guess we can wait to see if/when that happens before we decide where it would belong.

**Comment by jamesgorrie** [comment]
This is also for us to define - do we have content that can only live as a slice?
I imagine we would start with that and then try to export it out as a non-sliced component.

**Pull request merged**